src/perftest_parameters.c

```diff
@@ -2873,6 +2907,11 @@ void ctx_print_test_info(struct perftest_parameters *user_param)
 	printf(" Connection type : %s\t\tUsing SRQ      : %s\n", connStr[user_param->connection_type], user_param->use_srq ? "ON"  : "OFF");
 	#ifdef HAVE_RO
 	printf(" PCIe relax order: %s\n", user_param->disable_pcir ? "OFF"  : "ON");
+	if ((check_pcie_relaxed_ordering_compliant() == false) &&
+	    (user_param->disable_pcir == 0)) {
+		printf(" WARNING: CPU is not PCIe relaxed ordering compliant.\n");
+		printf(" WARNING: You should disable PCIe RO for both server and clinet.\n");
```

Comment:
Please, add to the warning the flag name --disable_pcie_relaxed, so user doesn't need to look through the help

Reply:
Yes, I will do it.